docs/src/main/tut/cookbook.md

```diff
@@ -351,9 +351,33 @@ val policy: Cors.Policy = Cors.Policy(
 val corsService: Service[Request, Response] = new Cors.HttpFilter(policy).andThen(service)
 ```
 
+### Creating a Finagle Client
```

Comment:
Can you also add a TOC item for this? I don't think we generate automatically.

Reply:
will do